warehouse/integrations/deltalake/deltalake.go

```diff
@@ -1075,15 +1068,13 @@ func GetDatabricksVersion() (databricksBuildVersion string) {
 
 	conn, err := grpc.DialContext(ctx, connectorURL, grpc.WithTransportCredentials(insecure.NewCredentials()))
 	if err != nil {
-		pkgLogger.Errorf("Error while creating grpc connection to databricks with error: %s", err.Error())
 		databricksBuildVersion = "Unable to create grpc connection to databricks."
```

Comment:
Why are we not returning error separately and making use of the same variable?

Reply:
This is some legacy code that is used during the `/version` call. If the API fails, we are just logging it.

Since we moved out the https://github.com/rudderlabs/rudder-server/pull/3148, we will remove this in the next version.